web/pgadmin/static/js/SchemaView/DataGridView.jsx

```diff
@@ -582,6 +589,18 @@ export default function DataGridView({
 
   const isResizing = _.flatMap(table.getHeaderGroups(), headerGroup => headerGroup.headers.map(header=>header.column.getIsResizing())).includes(true);
 
+  const virtualizer = useVirtualizer({
+    count: rows.length,
+    getScrollElement: () => tableRef.current,
+    estimateSize: () => 42,
+    measureElement:
+      typeof window !== 'undefined' &&
+        navigator.userAgent.indexOf('Firefox') === -1
+        ? element => element?.getBoundingClientRect().height
+        : undefined,
+    overscan: 10,
```

Comment:
Can we make `estimateSize` to configuration, and overscan as well?

Reply:
We can make `overscan` as config. `estimateSize` is the height of row of DataGridView which is roughly 42px, won't get much advantage by making it config based. Anyway, it is just an estimate - most likely to be ignored. Rows will be dynamically measured using `measureElement`.